src/lib/models/reflections/container.ts

```diff
@@ -38,10 +39,10 @@ export class ContainerReflection extends Reflection {
      * @param callback  The callback function that should be applied for each child reflection.
      */
     traverse(callback: TraverseCallback) {
-        if (this.children) {
-            this.children.slice().forEach((child: DeclarationReflection) => {
-                callback(child, TraverseProperty.Children);
-            });
+        for (const child of toArray(this.children).slice()) {
```

Comment:
Do you need to slice if you are already doing `toArray`?

Reply:
Good catch, I assumed `toArray` just returned the input if it was an array, but apparently it copies the array. Will remove.